sound/soc/sof/ipc3-dtrace.c

```diff
@@ -172,7 +174,9 @@ static int ipc3_trace_update_filter(struct snd_sof_dev *sdev, int num_elems,
 		dev_err(sdev->dev, "enabling device failed: %d\n", ret);
 		goto error;
 	}
+#if !ZEPHYR_HDA_DMA_LOGGING_BACKEND
```

Comment:
that's not going to work for CI, we would need a kernel parameter instead of a hard-coded build-time selection.